cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -181,6 +202,22 @@ def _get_entrance_exam(request, course_key):  # pylint: disable=W0613
         return HttpResponse(status=404)
 
 
+def update_entrance_exam(request, course_key, exam_data):
+    """
+    Operation to update course fields pertaining to entrance exams
+    The update operation is not currently exposed via the API
+    """
+    store = modulestore()
+    course = store.get_course(course_key)
+    if course is None:
+        return HttpResponse(status=400)
+
+    metadata = exam_data
+    CourseMetadata.update_from_dict(metadata, course, request.user)
+
+    return HttpResponse(status=200)
```

Comment:
This method returns values, but I don't think it should (the return values are ignored by the only place that calls the method).